test/PHPMailerTest.php

```diff
@@ -10,47 +10,50 @@
  * @license   http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License
  */
 
-namespace PHPMailer\PHPMailer;
+namespace PHPMailer\Test;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPUnit\Framework\TestCase;
 
 /**
  * PHPMailer - PHP email transport unit test class.
  */
-class PHPMailerTest extends \PHPUnit_Framework_TestCase
+final class PHPMailerTest extends TestCase
 {
     /**
      * Holds the PHPMailer instance.
      *
      * @var PHPMailer
      */
-    public $Mail;
+    private $Mail;
 
     /**
      * Holds the SMTP mail host.
      *
      * @var string
      */
-    public $Host = '';
+    private $Host = '';
 
     /**
      * Holds the change log.
      *
      * @var string[]
      */
-    public $ChangeLog = [];
+    private $ChangeLog = array();
```

Comment:
@Synchro `array()` long syntax is here on purpose so you can see how the build fails